cli/config/config.go

```diff
@@ -1,21 +1,26 @@
 package config
 
 import (
+	"fmt"
+
 	"github.com/ilyakaznacheev/cleanenv"
 	"github.com/spf13/cobra"
 )
 
 // Args expose available global args for cli command
 type Args struct {
-	ConfigPath      string
+	// ConfigPath is a path to main configuration file.
+	ConfigPath string
+	// ShareConfigPath is an additional config file (path) that (if present) will overwrite
+	// configuration supplied from config file at ConfigPath.
 	ShareConfigPath string
 }
```

Comment:
Perhaps we could add another comment line explaining why we might need 2 config files (not just 1).